roll/websockets.py

```diff
@@ -0,0 +1,147 @@
+import asyncio
+import types
+
+from websockets import handshake, WebSocketCommonProtocol, InvalidHandshake
+from websockets import ConnectionClosed  # exposed for convenience
+
+from roll import ProtocolUpgrade
+
+
+def create_websocket(
+        transport, subprotocol,
+        websocket_timeout=5,
```

Comment:
It looks to be difficult to override these settings without copy/pasting the whole `WebsocketProtocol.__call__` method, maybe we should define directly constants attached to the `WebsocketProtocol` class for easier configuration?

Reply:
Yes, i could not yet decide where they should live... The WebsocketProtocol would be the obvious choice.